packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -36,7 +36,37 @@ export function createClassFeaturePlugin({
   feature,
   loose,
   manipulateOptions,
+  // TODO(Babel 8): Remove the default falue
+  api = { assumption: () => {} },
 }) {
+  const setPublicClassFields = api.assumption("setPublicClassFields");
```

Comment:
This will throw on `@babel/core 7.12 + @babel/plugin-class-properties 7.13` because `api` is defined but `api.assumption` is not provided.